src/main/java/dinkplugin/message/DiscordMessageHandler.java

```diff
@@ -107,7 +125,9 @@ private void sendToMultiple(Collection<HttpUrl> urls, MultipartBody.Builder reqB
         urls.forEach(url -> sendMessage(url, reqBodyBuilder, 0));
     }
 
+    @SneakyThrows
```

Comment:
Why is this necessary now?

Reply:
`CountdownLatch#await` throws `InterruptedException`. Since `await` is only called for test cases, I think it's adequate to not add the checked exception to the method declaration of `sendMessage`